R/restrict_derivation.R

```diff
@@ -12,7 +12,7 @@
 #'   The function must provide the `dataset` argument and all arguments
 #'   specified in the `params()` objects passed to the `arg` argument.
 #'
-#'   Please note that it is not possible to specify `{dplyr}`
+#'   Please note that it is not advised to specify `{dplyr}`
```

Comment:
@bms63 Was it this that you meant?

Reply:
Umm...is this saying not to do `dplyr::mutate()` ?

Reply:
Well I'm not quite sure the origin story of this one, but all we wanted to do was loosen up the restriction such that we allow functions that don't necessarily have "dataset" as an argument

Reply:
@bundfussr any memory of this one?

Reply:
As far as I remember this sentence was added after the last clean-up to point out that `dplyr` functions can not be used. As this is changed now, we should remove the sentence completely.